https/users.go

```diff
@@ -37,14 +38,30 @@ func validateUsers(configPath string) error {
 }
 
 type userAuthRoundtrip struct {
-	tlsConfigPath string
-	handler       http.Handler
-	logger        log.Logger
+	tlsConfigPath   string
+	handler         http.Handler
+	logger          log.Logger
+	failuresCounter prometheus.Counter
+}
+
+func (u *userAuthRoundtrip) instrument(r prometheus.Registerer) {
+	u.failuresCounter = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Namespace: "prometheus_toolkit",
+			Subsystem: "https",
```

Comment:
https and basic auth are not the same thing

In general, it's better to have the metric name as a single string literal as it makes grepping and determining what the actual metric name is easier.

Reply:
I have renamed the metric.  However I think that keeping namespace separated is better, it that's fine for you.

Reply:
I preferred it named based on the package name.

Reply:
Should we change the package name?